### PR TITLE
add xhub

### PR DESCRIPTION
**Pull request by nschloe**
- [x] I have read, and re-read the [Contributing Guidelines](https://github.com/chartjs/awesome/blob/master/CONTRIBUTING.md)
- [x] I have searched to ensure the suggested item doesn't exist on this list
- [x] This PR contains only one item

### Please Describe Your Addition

[xhub](https://github.com/nschloe/xhub), a browser extension that lets you use Chart.js (and other nice things) on GitHub pages. I added a new section _Other_ because it wouldn't fit anywhere else, but I'll be happy to reorganize.

### CODE REVIEW

**Comment by kurkle** [comment]
This would need to be rebased. @simonbrunel @etimberg any thoughts about the section and/or if this is something accepted here?

**Comment by etimberg** [comment]
I think another section is fine

File: README.md [hunk 1]
```diff
@@ -82,3 +83,7 @@ In addition, many plugins can be found on the [npm registry](https://www.npmjs.c
 - [nova-chartjs](https://github.com/coroo/nova-chartjs) - Laravel Nova
 - [quickchart](https://github.com/typpo/quickchart) - Web API for static charts
 - [wicked-charts](https://github.com/adessoAG/wicked-charts) - Java
+
+
```
Comment (simonbrunel) [inline README.md]: Inconsistent extra line.

File: README.md [hunk 2]
```diff
@@ -82,3 +83,7 @@ In addition, many plugins can be found on the [npm registry](https://www.npmjs.c
...
+### Other
```
Comment (simonbrunel) [inline README.md]: Wrong sectioning, should be `##`

File: README.md [hunk 3]
```diff
@@ -82,3 +83,7 @@ In addition, many plugins can be found on the [npm registry](https://www.npmjs.c
...
+ - [xdoc](https://github.com/nschloe/xdoc) - Browser extension for using Chart.js on GitHub
```
Comment (simonbrunel) [inline README.md]: `nschloe/xdoc` redirects to `nschloe/xhub` where there is no mention of "xdoc". Should rename the link to `xhub`.

**Review by simonbrunel — CHANGES_REQUESTED** [review]
A new section is fine but I wouldn't call it "other". Maybe something like "Tools" is better?

**Comment by nschloe** [comment]
I've addressed the comment and reformatted the entry.

**Review by LeeLenaleee — APPROVED** [review]

**Review by simonbrunel — APPROVED** [review]

**Review by kurkle — APPROVED** [review]

**Pull request merged**